DesignSafe-CI/portal
# task/DES-1225 - Google Dataset Search

## Overview: ##
Added metadata to publication template for Google Dataset search.

## PR Status: ##


## Related Jira tickets: ##

* [DES-1225](https://jira.tacc.utexas.edu/browse/DES-1225)

## Summary of Changes: ##
Pulled data from publication model and ElasticSearch, and dumped it into a JSON, which is then put into the template for publication.

## Testing Steps: ##
1. Look at each type of publication, and check the HTML template (cmd + option + u)
2. Look for the tag <script type="application/ld+json">
3. Make sure the data is there, and that it passes a sanity check.
4. Insert publication's URL into Google's rich results test: https://search.google.com/test/rich-results
 

## UI Photos:

## Notes: ##
